environment.yml

```diff
@@ -1,32 +1,31 @@
 name: simulate_expression_compendia
 channels:
 - conda-forge
+- r
+- bioconda
 dependencies:
-- conda-forge::python=3.5.5
-- conda-forge::keras=2.1.6
-- conda-forge::tensorboard=1.10.0
-- conda-forge::tensorflow=1.10.0
-- conda-forge::pandas=0.22.0
+- r::r-base=3.6.0
+- bioconda::bioconductor-limma
+- bioconda::bioconductor-sva
+- conda-forge::python=3.7
+- conda-forge::keras
+- conda-forge::tensorflow=1
+- conda-forge::pandas=0.24.2
 - conda-forge::seaborn=0.8.1
-- conda-forge::numpy=1.15.2
-- conda-forge::numexpr=2.6.4
-- conda-forge::scikit-learn=0.19.1
+- conda-forge::numexpr
+- conda-forge::scikit-learn
 - conda-forge::scipy=1.1.0
-- conda-forge::hdf5=1.10.1
+- conda-forge::hdf5
 - conda-forge::umap-learn
 - conda-forge::plotnine
-- conda-forge::ggplot=0.11.5
-- conda-forge::graphviz=2.38.0
-- conda-forge::python-graphviz=0.8.4
-- conda-forge::ipykernel=4.8.1
-- conda-forge::ipywidgets=6.0.0
-- conda-forge::widgetsnbextension=2.0.0
-- conda-forge::nb_conda_kernels=2.1.0
+- conda-forge::ipykernel=5.1.1
+- conda-forge::nb_conda_kernels
 - conda-forge::gxx_linux-64
-- conda-forge::rpy2=2.8
+- conda-forge::rpy2
 - conda-forge::libiconv=1.15
 - conda-forge::joblib=0.13.2
-- pip=9.0
+- anaconda::protobuf
+- pip=19.2.1
 - pip:
   - torch>=0.4.0
   - matplotlib==3.0.0
```

Comment:
Don't forget to add ponyo to your environment file. You could do a pip install pointing to a github link for now, then update it once ponyo is added to pypi

Reply:
Currently I am manually pip installing ponyo into this conda environment because ponyo is not uploaded to pypi just yet. I plan to add ponyo to this yml file once it is.